internal/identity/google.go

```diff
@@ -8,7 +8,7 @@ import (
 	"net/http"
 	"net/url"
 
-	oidc "github.com/pomerium/go-oidc"
+	oidc "github.com/coreos/go-oidc"
```

Comment:
@desimone I realized there is a change in the go-oidc package, would you be kind to let me know what prompted this change?

Reply:
There were initially some changes between the upstream repo and our fork but those changes have since converged so it made sense to revert back to upstream.